homeassistant/components/modbus/sensor.py

```diff
@@ -175,9 +173,9 @@ async def async_setup_platform(
                 continue
         if CONF_HUB in entry:
             # from old config!
-            hub: ModbusHub = hass.data[MODBUS_DOMAIN][entry[CONF_HUB]]
+            hub = hass.data[MODBUS_DOMAIN][entry[CONF_HUB]]
```

Comment:
Same here.

Reply:
Same as above, when you read your suggestion, you would not expect the second instance to inherit the type from the first instance.

I prefer to keep it consistent, and no it will not be any type, but inherit the type from the assignment.